src/api/admin.ts

```diff
@@ -0,0 +1,114 @@
+import express from 'express'
+import fs from 'fs'
+
+type ServiceKey = {
+  id?: string,
+  key?: string, 
+  value?: string,
+  service?: string
+}
+
+type DB = {
+  serviceKeys: Array<ServiceKey>
+}
```

Comment:
could you put types into a separate folder? `import {ServiceKey, DB} from './types/admin.ts'`